src/components/CodeViewDefaultBlock.tsx

```diff
@@ -12,6 +12,9 @@ const CodeViewDefaultBlock = ({
 }: CodeViewDefaultBlockProps) => {
   const intl = useIntl();
   const ledPattern = makecodeIcons[icon];
+  const iconName = intl.formatMessage({
+    id: `led-icon-option-${icon.toLocaleLowerCase()}`,
```

Comment:
I doubt we want locale-specific lowercasing here.

Reply:
[0862e4b](https://github.com/microbit-foundation/ml-trainer/pull/462/commits/0862e4b2ef83ada6ebbd7a0142147fb3e1490077)